silhouette/Graphtec.py

```diff
@@ -138,11 +138,13 @@
 # Negative Acnoledge - Returns device tool setup
 CMD_NAK = '\x15'
 
+SILHOUETTE_CAMEO4_TOOL_EMPTY = 0
 SILHOUETTE_CAMEO4_TOOL_BLADE = 1
 SILHOUETTE_CAMEO4_TOOL_AUTOBLADE = 2
 SILHOUETTE_CAMEO4_TOOL_DEEPCUT = 3
 SILHOUETTE_CAMEO4_TOOL_2MM_CRAFTBLADE = 4
```

Comment:
You can further add
```
SILHOUETTE_CAMEO4_TOOL_ROTARYBLADE = 5
```
here.

Also it might make sense to rename `DEEPCUT`to `DEEPCUTBLADE` to be consistent with `KRAFTBALDE` and `AUTOBLADE`

Reply:
Check